src/app/controllers/conference_controller.py

```diff
@@ -1,39 +1,28 @@
 from PyQt5.QtCore import QObject, pyqtSignal, pyqtSlot
 
-from src.app.services.odas.odas import Odas
-from src.app.services.recorder.audio.audio_writer import AudioWriter, WriterActions
 from src.app.services.videoprocessing.video_processor import VideoProcessor
 from src.app.services.sourceclassifier.source_classifier import SourceClassifier
 
 
 class ConferenceController(QObject):
 
     signalException = pyqtSignal(Exception)
-    signalAudioPositions = pyqtSignal(object)
     signalOdasState = pyqtSignal(bool)
-    signalRecordingState = pyqtSignal(bool)
     signalVideoProcessorState = pyqtSignal(bool)
     signalVirtualCamerasReceived = pyqtSignal(object)
     signalHumanSourcesDetected = pyqtSignal(object)
+    signalAudioPositions = pyqtSignal(object)
 
-    def __init__(self, parent=None):
+    def __init__(self, odasserver, parent=None):
         super(ConferenceController, self).__init__(parent)
 
-        self.__odas = Odas(hostIP='127.0.0.1', portPositions=10020, portAudio=10030, isVerbose=False)
-        self.__odas.start()
+        self.isOdasLiveConnected = False
+        self.videoProcessorState = False
 
-        self.__audioWriter = AudioWriter()
-        self.__audioWriter.changeWavSettings(outputFolder='', nChannels=4, nChannelFile=1, byteDepth=2, sampleRate=48000)
-        self.__audioWriter.start()
-        self.__isRecording = False
-        self.__odasLiveConnected = False
-
-        self.__audioWriter.signalException.connect(self.audioWriterExceptionHandling)
-        self.__odas.signalException.connect(self.odasExceptionHandling)
-        
-        self.__odas.signalAudioData.connect(self.audioDataReceived)
-        self.__odas.signalPositionData.connect(self.positionDataReceived)
-        self.__odas.signalClientsConnected.connect(self.odasClientConnected)
+        self.__odasserver = odasserver
+        self.__odasserver.signalException.connect(self.odasExceptionHandling)
+        self.__odasserver.signalPositionData.connect(self.positionDataReceived)
+        self.__odasserver.signalClientsConnected.connect(self.odasClientConnected)
 
         self.__videoProcessor = VideoProcessor()
```

Comment:
We should not create two videoprocessors (in conference controller and in recording controller). It should be passed by the main window.